test/lib/sdkdriver/src/testdriver/browser/selenium.ts

```diff
@@ -0,0 +1,45 @@
+import { WebDriver } from 'selenium-webdriver';
+import { createWebDriver } from '../../util';
+import BrowserDriver from './base';
+
+/**
+ * Selenium browser process driver.
+ * @class
+ * @classdesc A {@link SeleniumBrowserDriver} manages a Selenium WebDriver
+ *   browser process where the cross-browser test is running.
+ */
+export default class SeleniumBrowserDriver extends BrowserDriver {
+  private readonly _webDriver: WebDriver;
+
+  /**
+   * Constructor.
+   * @param {"chrome" | "firefox"} browser
+   * @param {string} sdkUrl
+   */
+  constructor(browser: 'chrome' | 'firefox', sdkUrl: string) {
+    super({
+      browser,
+      host: 'localhost',
+      params: { sdkUrl }
+    });
+    this._webDriver = createWebDriver(browser);
+  }
+
+  /**
+   * Start the browser process.
+   * @param {string} url - URL for running the SDK setup code
+   * @returns {Promise<void>}
+   */
+  async startBrowser(url: string): Promise<void> {
```

Comment:
Actually, I guess this could be made `protected`?

Reply:
@markandrus TypScript does not allow `abstract` methods to be `protected`. So I have to override with public methods.